### PR TITLE
added 1CAK Presence

### PR DESCRIPTION
**Pull request by Allvaa**
Capture
Capture2
Capture3
Capture4

### CODE REVIEW

File: 1CAK/presence.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        presenceData.details = "Legendary Page";
+    } else if (document.location.pathname === "/friends") {
+        presenceData.details = "Friends Page";
+    } else if (document.location.pathname === "/rules") {
+        presenceData.details = "Reading the rules";
+    } else if (document.location.pathname === "/notifications") {
+        presenceData.details = "Notifications Page";
+    } else if (document.location.pathname === "/weeklytop" || document.location.pathname === "/alltimetop") {
+        presenceData.details = "Top Users";
+    } else if (document.location.pathname === "/preferences" || document.location.pathname === "/privacy_setting") {
```
Comment (Timeraa) [inline 1CAK/presence.ts]: Maybe use switch case here?

Comment (Allvaa) [reply 1CAK/presence.ts]: should i? i already sleep, may i will do it tomorrow

Comment (Allvaa) [reply 1CAK/presence.ts]: k done.

File: 1CAK/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "service": "1CAK",
+    "description": {
+        "en": "1CAK is an Indonesian entertainment site that provides images and videos uploaded by users. This site is also known as entertainment provider through Internet meme. 1CAK users can select and comment on images. Popular images appear on the main website.[2]"
+    },
+    "url": [
+        "1cak.com"
+    ],
+    "version": "1.0",
+    "logo": "https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/1CAK-logo.png/170px-1CAK-logo.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/504074689182957569/631174212152852490/unknown.png",
```
Comment (Bas950) [inline 1CAK/dist/metadata.json]: Thumbnail is of wrong service

Comment (Bas950) [reply 1CAK/dist/metadata.json]: nvm this was changed later

**Pull request merged**